k8s/crd/templates/v20.yaml

```diff
@@ -0,0 +1,2548 @@
+{{ if eq .Values.version "v20" }}
```

Comment:
How was this file generated?
Might be worth documenting somewhere the process to add support for a new API version.

Reply:
Thanks!